substra/cli/interface.py

```diff
@@ -74,14 +74,26 @@ def display(res):
     print(res)
 
 
-# TODO profile, config, json and verbose options should be handled in a single
-#      decorator to populate a GlobalOption object stored in the context
+class GlobalConf:
+    profile = None
+    config = None
+    user = None
+    verbose = None
+    expand = None
```

Comment:
You're missing `output_format`.

Reply:
It is not required for all methods where we use click_option_common. For example, it is not used in add_data_sample, describe or download, that's why it isn't in the GlobalConf class

Reply:
The attributes we're listing here are not really useful : since we use getattr/setattr, there's no need for them. Their purpose is more to inform the developer of which attributes may be available. And for that purpose, I think that the `output_format` should be added.

Reply:
Ok, I'll add it!

Reply:
👍, it's better to be explicit.

I'm unresolving this comment as `output_format` is missing.